streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -18,7 +18,7 @@
 package org.apache.kafka.streams;
 
 import org.apache.kafka.common.serialization.Serdes;
-import org.apache.kafka.streams.integration.utils.EmbeddedSingleNodeKafkaCluster;
+import org.apache.kafka.streams.integration.utils.EmbeddedSingleZKKafkaCluster;
```

Comment:
Could we just rename to `EmbeddedKafkaCluster` for simplicity, since we will not probably have multi-node ZK in the future.